api/src/main/java/com/codeforcommunity/dto/EventReturn.java

```diff
@@ -0,0 +1,22 @@
+package com.codeforcommunity.dto;
+
+import java.time.LocalDateTime;
+
+public class EventReturn {
+  // doesnt make sense that we would ever set these values after construction.
+  // favor immutibility.
+  public final int id;
```

Comment:
Make these private and add public getters. It seems redundant but if these get requests need to be refactored then having the fields only be accessed through methods saves us a lot of headache